lib/reversetunnel/agent.go

```diff
@@ -170,12 +294,12 @@ func (a *Agent) connect() (conn *ssh.Client, err error) {
 }
 
 func (a *Agent) proxyAccessPoint(ch ssh.Channel, req <-chan *ssh.Request) {
-	log.Debugf("[HA Agent] proxyAccessPoint")
+	a.Debugf("proxyAccessPoint")
```

Comment:
Not in your PR but below in this function `conn.Close()` is called twice.

Reply:
it is actually on purpose, to make sure the other connection leg will close, because it could get stuck on io.Copy otherwise